2010-09-16      Jerome Lambourg <lambourg@adacore.com>

	* glib/gconvert.c: remove explicit check for GNU libiconv as the
        Snow Leopart system library is not GNU but still compatible.


--- glib/gconvert.c.old	2010-01-22 18:27:30.000000000 +0100
+++ glib/gconvert.c	2010-01-22 18:28:36.000000000 +0100
@@ -51,9 +51,6 @@
 #if defined(USE_LIBICONV_GNU) && !defined (_LIBICONV_H)
 #error GNU libiconv in use but included iconv.h not from libiconv
 #endif
-#if !defined(USE_LIBICONV_GNU) && defined (_LIBICONV_H)
-#error GNU libiconv not in use but included iconv.h is from libiconv
-#endif
 
 #include "galias.h"
 
